frontend/src/components/pages/agents/details/chat/agent-chat-tab.tsx

```diff
@@ -52,9 +54,13 @@ export const AgentChatTab = ({ agent }: AgentChatTabProps) => {
 
   return (
```

Comment:
Let's add some form of agent missing indicator
```suggestion
if (!agent) {
return (
<div>Agent is not available right now</div>
)
}
```